src/main/java/io/cdap/plugin/salesforce/parser/SalesforceQueryVisitor.java

```diff
@@ -57,6 +59,30 @@ public SObjectDescriptor visitStatement(SOQLParser.StatementContext ctx) {
       throw new SOQLParsingException("SOQL must have at least one field: " + ctx.getText());
     }
 
+    if (ctx.fromStatement().GROUP() == null) {
+      List<String> aliasedFields = fields.stream()
+        .filter(fieldDescriptor -> fieldDescriptor.getFunctionType() == SalesforceFunctionType.UNDEFINED)
+        .filter(SObjectDescriptor.FieldDescriptor::hasAlias)
+        .map(SObjectDescriptor.FieldDescriptor::getFullName)
+        .collect(Collectors.toList());
+
+      if (!aliasedFields.isEmpty()) {
```

Comment:
I see. Can you add a comment? 

In general, it would be good to add comments that relate to how the Salesforce API is defined or how it behaves. You now know a lot of Salesforce specific information that the next person who touches the code might not know.